hack/generate-yamls.sh

```diff
@@ -95,7 +96,13 @@ ko resolve ${KO_YAML_FLAGS} -f config/brokers/mt-channel-broker/ | "${LABEL_YAML
 # Create in memory channel yaml
 ko resolve ${KO_YAML_FLAGS} -Rf config/channels/in-memory-channel/ | "${LABEL_YAML_CMD[@]}" > "${EVENTING_IN_MEMORY_CHANNEL_YAML}"
 
-all_yamls=(${EVENTING_CORE_YAML} ${EVENTING_CRDS_YAML} ${EVENTING_MT_CHANNEL_BROKER_YAML} ${EVENTING_IN_MEMORY_CHANNEL_YAML} ${EVENTING_YAML})
+# Create eventing TLS yaml
+ko resolve ${KO_YAML_FLAGS} -Rf config/tls/ \
+  -Rf config/channels/in-memory-channel-tls/ \
+  -Rf config/brokers/mt-channel-broker-tls/ \
+  | "${LABEL_YAML_CMD[@]}" > "${EVENTING_TLS_YAML}"
```

Comment:
```suggestion
ko resolve "${KO_YAML_FLAGS}" -Rf config/tls/ \
  -Rf config/channels/in-memory-channel-tls/ \
  -Rf config/brokers/mt-channel-broker-tls/ \
  | "${LABEL_YAML_CMD[@]}" > "${EVENTING_TLS_YAML}"
```

For consistency, as that is how its done at other places in the PR